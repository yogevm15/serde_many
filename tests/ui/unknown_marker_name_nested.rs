use serde_many::{DeserializeMany, SerializeMany};

struct Default;

#[derive(SerializeMany, DeserializeMany)]
#[serde_many(default = "Default")]
#[serde(default(transparent), foo(aaaa))]
struct A(i32);

fn main() {}
