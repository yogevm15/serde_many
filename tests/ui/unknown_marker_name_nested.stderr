error: Unknown marker name, have you forgotten to add it to `#[serde_many(...)]`?
 --> tests/ui/unknown_marker_name_nested.rs:7:31
  |
7 | #[serde(default(transparent), foo(aaaa))]
  |                               ^^^
